crates/ide-assists/src/handlers/inline_type_alias.rs

```diff
@@ -84,6 +84,9 @@ pub(crate) fn inline_type_alias_uses(acc: &mut Assists, ctx: &AssistContext<'_>)
             for (file_id, refs) in usages.into_iter() {
                 inline_refs_for_file(file_id, refs);
             }
+
+            builder.edit_file(ctx.file_id());
+            builder.delete(ast_alias.syntax().text_range());
```

Comment:
You'll have to extract the edits for `ctx.file_id()` out of `usages` and do the removal/replacement in one go like is done in `inline_call`. reason being if you `edit_file` for file `0`, then for `1` and then for `0` again you discard the previous edits.

Reply:
Could you please check my latest changes?